### ISSUE TITLE
Search input to change to input="search"

### ISSUE THREAD

**Issue by coliff** [squidfunk/mkdocs-material#309]
## Description

The Search input is currently type="text". Changing it to type="search" would be more semantically correct and may help with accessibility. The change should be simple. Chrome adds an additional 'close' button on search inputs though, to set this to be hidden we can add the following CSS

`input[type=search]::-webkit-search-cancel-button{-webkit-appearance:none}`

I've tested this locally and in production and have encountered no issues (Chrome, Safari, Firefox, Edge and IE11)

**Comment by squidfunk** [comment]
I would like to evaluate this in a complete a11y context, improving it for the whole theme and taking ARIA roles etc. into account. However, this may not work in Edge or Opera, because they may user other vendor prefixes or do the rendering otherwise. It's to risky to change at the moment, because there are no tests that capture the look and behavior of this element.

**Comment by squidfunk** [comment]
Closing this for now, as accessibility is a big topic that needs to be addressed in the future, but there's currently no time for that.
